File: CPP/344.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
      std::reverse(s.begin(), s.end());
    }
};

class Solution {
public:
    void reverseString(vector<char>& s) {
        for (int i = 0; i < s.size()/2; ++i){
            std::swap(s[i], s[s.size() - 1 - i]);
        }
    }
};

File: Python/344.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        s.reverse()

class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        for index in range(0, len(s)//2):
            temp = s[index]
            s[index] = s[len(s)-1-index]
            s[len(s) -1 -index] = temp
